### ISSUE TITLE
Feature Request: Morphology of Apparatus

### ISSUE THREAD

**Issue by jcuenod** [morphgnt/sblgnt#48]
For searching it would be pretty useful to have the critical data parsed - I don't know what you think of that idea?

**Comment by jtauber** [comment]
Yes, I'd like to get to it. I'd actually like to make sure all forms in a range of base texts and apparatus are parsed.
